evm_loader/lib/src/types/tracer_rocks_db.rs

```diff
@@ -0,0 +1,162 @@
+use abi_stable::traits::IntoOwned;
+use async_trait::async_trait;
+use jsonrpsee::core::client::ClientT;
+use jsonrpsee::core::Serialize;
+use jsonrpsee::rpc_params;
+use jsonrpsee::ws_client::{WsClient, WsClientBuilder};
+use serde_json::{from_slice, from_str};
+use std::str::FromStr;
+use std::sync::Arc;
+// pub type RocksDbResult<T> = std::result::Result<T, anyhow::Error>;
+use crate::config;
+use solana_sdk::signature::Signature;
+use solana_sdk::{
+    account::Account,
+    clock::{Slot, UnixTimestamp},
+    pubkey::Pubkey,
+};
+use tracing::info;
+
+#[derive(Clone, Serialize)]
+pub struct AccountParams {
+    pub pubkey: Pubkey,
+    pub slot: u64,
+    pub tx_index_in_block: Option<u64>,
+}
+
+use crate::types::tracer_ch_common::{EthSyncStatus, RevisionMap};
+use crate::types::{DbResult, TracerDb};
+// use reconnecting_jsonrpsee_ws_client::{Client, CallRetryPolicy, rpc_params, ExponentialBackoff};
+
+#[derive(Clone, Debug)]
+pub struct RocksDb {
+    #[allow(dead_code)]
+    url: String,
+    client: Arc<WsClient>,
+}
+
+impl RocksDb {
+    #[must_use]
+    pub async fn new() -> Self {
+        let config = config::load_rocks_db_config_from_environment();
```

Comment:
I suggest explicitly passing parameters to the `new()` method. In this case, we can use `RocksDb` more flexibly. The same about `ClickHouseDb`